app/test/integration-test/src/test/resources/io/syndesis/test/itest/mail/SendMail-export/model.json

```diff
@@ -1,2480 +1 @@
onnectorCustomizers":["io.syndesis.connector.sql.customizer.SqlStartStoredConnectorCustomizer"],"inputDataShape":{"kind":"none"},"outputDataShape":{"kind":"json-schema"},"propertyDefinitionSteps":[{"description":"Select the stored procedure.","name":"Procedure name","properties":{"procedureName":{"componentProperty":true,"deprecated":false,"displayName":"Procedure name","group":"producer","javaType":"java.lang.String","kind":"path","labelHint":"Name of the stored procedure.","required":true,"secret":false,"type":"select"},"schedulerExpression":{"defaultValue":"60000","deprecated":false,"displayName":"Period","group":"consumer","javaType":"long","kind":"parameter","labelHint":"Delay between scheduling (executing).","required":false,"secret":false,"type":"duration"},"template":{"componentProperty":false,"deprecated":false,"displayName":"Template","group":"producer","javaType":"java.lang.String","kind":"path","labelHint":"Stored Procedure template to perform.","required":true,"secret":false,"type":"hidden"}}}]},"id":"sql-stored-start-connector","name":"Periodic stored procedure invocation","pattern":"From","tags":["dynamic"]}],"connectorCustomizers":["io.syndesis.connector.sql.customizer.DataSourceCustomizer"],"dependencies":[{"id":"io.syndesis.connector:connector-sql:2.0-SNAPSHOT","type":"MAVEN"},{"id":"jdbc-driver","type":"EXTENSION_TAG"}],"description":"Invoke SQL to obtain, store, update, or delete data.","icon":"assets:sql.svg","id":"sql","name":"Database","properties":{"password":{"componentProperty":true,"deprecated":false,"displayName":"Password","group":"security","javaType":"java.lang.String","kind":"property","label":"common,security","labelHint":"Password for the database connection.","order":3,"required":true,"secret":true,"type":"string"},"schema":{"componentProperty":true,"deprecated":false,"displayName":"Schema","group":"common","javaType":"java.lang.String","kind":"property","label":"common","labelHint":"Database schema.","order":4,"required":false,"secret":false,"type":"string"},"url":{"componentProperty":true,"deprecated":false,"displayName":"Connection URL","group":"common","javaType":"java.lang.String","kind":"property","labelHint":"JDBC URL of the database.","order":1,"required":true,"secret":false,"type":"string"},"user":{"componentProperty":true,"deprecated":false,"displayName":"Username","group":"common","javaType":"java.lang.String","kind":"property","labelHint":"Username for the database connection.","order":2,"required":true,"secret":false,"type":"string"}},"tags":["verifier"],"version":16},"connectorId":"sql","description":"Connection to SampleDB","icon":"assets:sql.svg","id":"5","isDerived":false,"name":"PostgresDB","tags":["sampledb"],"uses":0},"id":"-M4y0fphoj9AnQ4QczWE","metadata":{"configured":"true"},"stepKind":"endpoint"}],"tags":["5","i-LhL2-ccjv5WqGyjCaVVz","webhook"],"type":"PRIMARY"}],"id":"i-M4y1EddlCY5ii3d0hYCz","name":"SendMail","tags":["email-send","sql","webhook"],"updatedAt":1586961902021,"version":10}}}
```

Comment:
Could you please format the new Json so the github diff is able to mark changes in a better way?

Reply:
Yes, make sense.

The change was because I redid it to add the new mandatory field on email connector (the "from")